library/aos_version.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
```

Comment:
Well, the Ansible docs say:

http://docs.ansible.com/ansible/playbooks_roles.html
> Roles dependencies are always executed before the role that includes them, and are recursive. By default, roles can also only be added as a dependency once - if another role also lists it as a dependency it will not be run again. This behavior can be overridden by adding allow_duplicates: yes to the meta/main.yml file.

I did not set `allow_duplicates`, so I understand `openshift_facts` should be run only once...

Puzzled :confused: